### ISSUE TITLE
textual display errors

### ISSUE THREAD

**Issue by harryparkdotio** [zeit/hyper#882]
![screen shot 2016-10-15 at 9 40 03 pm](https://cloud.githubusercontent.com/assets/11331772/19409490/21cc8d0c-9320-11e6-8174-627ac48bde7d.png)

This was just by installing Oh-my-zsh through curl on a fresh install of OS X El Capitan. Not too sure whats happening here but I've also seen similar looking things when using the immediately after using the nano command

**Comment by lightster** [comment]
@harryparkdotio Do you (or maybe zsh?) have a custom PS1 environment variable set? I was having similar graphical corruption that was caused by me improperly escaping color escapes in my PS1 env var. It annoyed me for a week before I decided to figure it out.

I was defining my PS1 as:
```sh
RESET=$(tput sgr0)
BOLD=$(tput bold)
BLACK=$(tput setaf 0)
RED=$(tput setaf 1)
GREEN=$(tput setaf 2)
YELLOW=$(tput setaf 3)
BLUE=$(tput setaf 4)
MAGENTA=$(tput setaf 5)
CYAN=$(tput setaf 6)
WHITE=$(tput setaf 7)

HOST='\h'
CWD='\w'
GIT='$(__git_ps1 " (%s)")'
TIME='\t'
PS1="\n[$GREEN$TIME $RED$HOST$WHITE:$BLUE$CWD$YELLOW$GIT$WHITE]"
PS1="$PS1\n    $PURPLE\$$RESET "
```

The colors should actually have escaped square brackets around them like so:
```sh
RESET="\[$(tput sgr0)\]"
BOLD="\[$(tput bold)\]"
BLACK="\[$(tput setaf 0)\]"
RED="\[$(tput setaf 1)\]"
GREEN="\[$(tput setaf 2)\]"
YELLOW="\[$(tput setaf 3)\]"
BLUE="\[$(tput setaf 4)\]"
MAGENTA="\[$(tput setaf 5)\]"
CYAN="\[$(tput setaf 6)\]"
WHITE="\[$(tput setaf 7)\]"

HOST='\h'
CWD='\w'
GIT='$(__git_ps1 " (%s)")'
TIME='\t'
PS1="\n[$GREEN$TIME $RED$HOST$WHITE:$BLUE$CWD$YELLOW$GIT$WHITE]"
PS1="$PS1\n    $PURPLE\$$RESET "
```

Perhaps that will help?

Good luck!

**Comment by harryparkdotio** [comment]
Thanks for the help, but I actually don't have any custom PS1 environment variables set in either my `.zshrc` or my hyper &rarr; preferences. Still experiencing these issues, including when using nano

![screen shot 2016-10-25 at 9 46 22 am](https://cloud.githubusercontent.com/assets/11331772/19666702/f2797216-9a97-11e6-98d2-ecc3ff4ce2d3.png)

**Comment by harryparkdotio** [comment]
This issue has been fixed in v1.0.0.
